.circleci/config.yml

```diff
@@ -134,7 +185,7 @@ commands:
             # https://github.com/rust-lang/rust/issues/49078
             # https://wiki.archlinux.org/index.php/Rust#Windows
             for lib in crt2.o dllcrt2.o libmsvcrt.a; do cp -v /usr/x86_64-w64-mingw32/lib/$lib ~/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-pc-windows-gnu/lib/; done
-            for lib in crt2.o dllcrt2.o libmsvcrt.a; do cp -v /usr/i686-w64-mingw32/lib/$lib ~/.rustup/toolchains/stable-i686-unknown-linux-gnu/lib/rustlib/i686-pc-windows-gnu/lib/; done
+            for lib in crt2.o dllcrt2.o libmsvcrt.a; do cp -v /usr/i686-w64-mingw32/lib/$lib ~/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/lib/rustlib/i686-pc-windows-gnu/lib/; done
```

Comment:
Why are we changing this line partially?

Reply:
It was wrong when it was first added in #776, but it never ran in CI since it used to only be used for deploying releases.  The first part of that path is the host architecture (which is always 64-bit), and the second is the target (which is 32-bit).

Reply:
Thanks for clarifying!